core/src/storage/device_old.py

```diff
@@ -0,0 +1,56 @@
+from micropython import const
+from typing import TYPE_CHECKING
+
+import storage
+import storage.cache
+
+if TYPE_CHECKING:
+    from typing_extensions import Literal
+
+
+# fmt: off
+# Keys:
+DEVICE_ID                  = const(0x00)  # bytes
+# MOVE to storage FIDO2
+U2F_COUNTER                = const(0x09)  # int
+# TODO: move to init
+INITIALIZED                = const(0x13)  # bool (0x01 or empty)
```

Comment:
Why keep these here? It should be possible to expose the constants in `obj_module!`.

Reply:
These are actually leftovers from the past, they are no longer needed anymore - currently all the storage handling is done in Rust.

Some other constants like `HOMESCREEN_MAXSIZE` really could be exposed, will try that